includes/admin/class-wc-admin-attributes.php

```diff
@@ -252,7 +252,7 @@ public static function edit_attribute() {
 
 		?>
 		<div class="wrap woocommerce">
-			<h1><?php _e( 'Edit attribute', 'woocommerce' ) ?></h1>
+			<h1><?php echo get_admin_page_title(); ?></h1>
```

Comment:
Here still display `Attributes` in the page and not `Edit attribute`.
So better no change this one.